lib/modules/dosomething/dosomething_user/dosomething_user.theme.inc

```diff
@@ -7,15 +7,16 @@
 /**
  * Returns markup for link to sign up for a campaign.
  *
- * @param string $label
- *   The text to display on the button.
+ * @param string $label - The text to display on the button.
+ * @param string $nid - The campaign node ID
+ * @param string $title - The campaign title
  * @return string
  */
-function dosomething_user_get_sign_up_link_markup($label, $nid) {
+function dosomething_user_get_sign_up_link_markup($label, $nid, $title) {
```

Comment:
Should we default to something for `$title`? Or are we requiring always passing the node title when calling the function?